db/schema.rb

```diff
@@ -45,6 +45,9 @@
     t.integer "councellor_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "issue_status", default: 0
```

Comment:
These fields were there already - the ones with _id. Since they were going to link to other tables.

Reply:
one of them was mis-named and looked mixed up between the parent and sub type so thought it would be best to create a new one and delete the old field if unnecessary. Basically was me just trying out migrations with rails to get a handle on how they work.